### PR TITLE
Add content for pipelines

### PR DESCRIPTION
**Pull request by AsfaMumtaz**

### CODE REVIEW

**Review by ayeshasiddiqa161 — DISMISSED** [review]
Approved

**Review by rasheedamir — CHANGES_REQUESTED** [review]
Will delivery engineers be kubeadmins? Why create new SCC? Why existing SCCs can't be used?

File: content/for-delivery-engineers/tutorials/preparing-env-for-pipeline-as-code/prepare-env.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+Apply it to the cluster. You can do this directly through the OpenShift console.
+  
+   ![SCC](images/allowpriv.png)
+
+## Create Role and RoleBinding
+
+1. Create a ClusterRole named *pipelines-scc-role*
+
+```yaml
+
```
Comment (karl-johan-grahn) [inline content/for-delivery-engineers/tutorials/preparing-env-for-pipeline-as-code/prepare-env.md:77]: you don't need the empty lines before and after the actual code

File: content/for-delivery-engineers/tutorials/preparing-env-for-pipeline-as-code/prepare-env.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+Apply it to the cluster. You can do this directly through the OpenShift console.
+  
+   ![SCC](images/allowpriv.png)
+
+## Create Role and RoleBinding
+
+1. Create a ClusterRole named *pipelines-scc-role*
+
+```yaml
```
Comment (karl-johan-grahn) [inline content/for-delivery-engineers/tutorials/preparing-env-for-pipeline-as-code/prepare-env.md:76]: You need to indent the yaml if you want it to appear as part of the first bullet

File: content/for-delivery-engineers/tutorials/preparing-env-for-pipeline-as-code/prepare-env.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    apiGroups:
+      - security.openshift.io
+    resources:
+      - securitycontextconstraints
+    resourceNames:
+      - tekton-pipelines-scc
+
+```
+
+Now let's create a RoleBinding for this:
```
Comment (karl-johan-grahn) [inline content/for-delivery-engineers/tutorials/preparing-env-for-pipeline-as-code/prepare-env.md:94]: bullet two, right?

File: content/for-delivery-engineers/tutorials/preparing-env-for-pipeline-as-code/prepare-env.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+     - persistentVolumeClaim
+     - projected
+     - secret
+   allowHostPID: false
+   allowHostNetwork: false
+   allowPrivilegeEscalation: true
+   allowedCapabilities: null
+```
+
+Apply it to the cluster. You can do this directly through the OpenShift console.
```
Comment (karl-johan-grahn) [inline content/for-delivery-engineers/tutorials/preparing-env-for-pipeline-as-code/prepare-env.md:68]: bullet two, right?

**Review by karl-johan-grahn — CHANGES_REQUESTED** [review]

File: content/for-delivery-engineers/tutorials/preparing-env-for-pipeline-as-code/prepare-env.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
+# Setting Up Service Account, SCC, Role, and RoleBinding for Tekton Pipelines
+
+In this tutorial, we will guide you through the process of setting up a service account, a Security Context Constraint (SCC), a role, and a role binding for Tekton Pipelines. These steps are essential for managing your pipeline's security and access within your Kubernetes/OpenShift cluster.
+
+## Prerequisites
+
+- Access to a Kubernetes or OpenShift cluster.
```
Comment (rasheedamir) [inline content/for-delivery-engineers/tutorials/preparing-env-for-pipeline-as-code/prepare-env.md:7]: What kind of access? Customer delivery engineers can't be clusteradmins; maybe SCA? but then you should validate as SCA and not kubeadmin and document as SCA and not kubeadmin

**Pull request closed without merging**